### DISCUSSION TITLE
[FEATURE] Is there any way to disable loading any comments from a reddit post using the archive command?

### DISCUSSION THREAD

**Discussion by arcticfly** [Serene-Arc/bulk-downloader-for-reddit#790] [Q&A]
- [x] I am requesting a feature.
- [x] I am running the latest version of BDfR
- [x] I have read the [Opening an issue](../../README.md#configuration)

## Description

I am trying to download a large number of reddit posts, load them into json objects, and parse the titles and post text. If I find a post that passes my filters, I want to send myself the permalink to that post. However, on some of these posts, the comments section is huge, which not only takes up more space on my computer but also increases the read/write time for going through these posts. I also suspect that it increases the download time. Is there any way to specify that no comments should be loaded when archiving a subreddit?

Here's my current opts file:

```
file_scheme: "{UPVOTES}_{REDDITOR}_{POSTID}_{DATE}"
limit: 5000
sort: top
make_hard_links: true
all_comments: false
subreddit:
  - startups
```

Apologies if there is an easy way to do this that I just missed! I've read through the README and haven't seen anything that matches my need yet!

**Comment by Serene-Arc** [comment]
Oops sorry, shouldn't have converted this to a discussion. I thought that this wasn't the normal behaviour but it is; the BDFR automatically gets the top level comments and then recurses down. This should be allowed to be disabled. #791 is the new issue
